lib/src/easy_ads.dart

```diff
@@ -1,8 +1,11 @@
+import 'dart:async';
```

Comment:
you didn't update the example app for how to use these events, no changelog, no read me update?